libsrc/atari/getdefdev.s

```diff
@@ -68,16 +69,23 @@ crvec:  jsr     $FFFF           ; will be set to crunch vector
         sta     __defdev
         iny
         lda     (DOSVEC),y
-        sta     __defdev+1
+done:   sta     __defdev+1
 
 ; Return pointer to default device
 
 finish: lda     #<__defdev
         ldx     #>__defdev
         rts
 
+; XDOS version
+
+xdos:   lda     XDEFDEV
+        bne     done
+
         .data
 
+crvec:	.byte	$4C,$FF,$FF     ; will be set to crunch vector
```

Comment:
What is the reason to not write 'jmp $FFFF' here?

The decision of mnemonics or .byte is totally independent from the segment. The cc65 code base already contains lots of places where mnemonics are used in the .data segment (and places where .byte is used in the .code segment).